tests/test_ilastik/test_workflows/testObjectClassificationGui.py

```diff
@@ -253,6 +252,13 @@ def impl():
                 for plugin in features
             }
 
+            # in order to also calculate neighborhood features, we need to set one
+            # margin to non-zero
+            for plugin in features:
+                for _, feature_descr in features[plugin].items():
+                    if "margin" in feature_descr:
+                        feature_descr["margin"] = (3, 3, 3)
```

Comment:
I'm not familiar with what the properties of the feature description mean, but why do we need to overwrite a value if it exists?

Reply:
yes, this setting is to be overridden by the UI (hence, manually in test). The existence of the "margin" key indicates that a feature is computed on the object + some neighbourhood around it. The logic will hopefully soon change, but for now it's like that - and for the tests to trigger the computation of those neighbourhood features, there needs to be a non-zero value (all "neighbourhood features" have a 0 there as default).